xlr/sdk/src/utils.ts

```diff
@@ -44,9 +56,23 @@ export function simpleTransformGenerator<
           };
         }
 
+        // need to walk generic tokens
         return {
           ...node,
           properties: { ...newObjectProperties },
+          genericTokens: isGenericNamedType(node)
+            ? node.genericTokens.map((token) => {
+                return {
+                  ...token,
+                  constraints: token.constraints
+                    ? walker(token.constraints, capability)
+                    : undefined,
+                  default: token.default
+                    ? walker(token.default, capability)
+                    : undefined,
+                };
+              })
+            : undefined,
```

Comment:
Could you avoid adding `genericTokens: undefined`?
```suggestion
          ...(isGenericNamedType(node)
            ? {genericTokens: node.genericTokens.map((token) => {
                return {
                  ...token,
                  constraints: token.constraints
                    ? walker(token.constraints, capability)
                    : undefined,
                  default: token.default
                    ? walker(token.default, capability)
                    : undefined,
                };
              })}
            : {),
```

Reply:
Good call, fixed.